clippy_lints/src/loops.rs

```diff
@@ -1684,6 +1711,27 @@ fn check_for_loop_over_map_kv<'tcx>(
     }
 }
 
+fn check_for_single_element_loop<'tcx>(
+    cx: &LateContext<'tcx>,
+    _: &'tcx Pat<'_>,
+    arg: &'tcx Expr<'_>,
+    _: &'tcx Expr<'_>,
+    _: &'tcx Expr<'_>,
```

Comment:
No need to add all arguments, if they are unused.

Reply:
Oh, I was just following the style of the other functions, removed!